Lib/multiprocessing/pool.py

```diff
@@ -596,7 +622,7 @@ def _help_stuff_finish(inqueue, task_handler, size):
             time.sleep(0)
 
     @classmethod
-    def _terminate_pool(cls, taskqueue, inqueue, outqueue, pool,
+    def _terminate_pool(cls, taskqueue, inqueue, outqueue, pool, pool_notifier,
```

Comment:
i would prefer to reuse the same variable name: pool_notifier => change_notifier.